1-exercises/B-todo-list/style.css

```diff
@@ -123,6 +123,14 @@ body {
 .red {
   background-color: red;
 }
+/* style from js */
+.cross {
+  text-decoration: line-through;
+}
+li {
+  font-size: 5rem;
```

Comment:
It's nice to see you having a go at some styling, just make sure if you're changing the font size to be this large, that it doesn't push the content off the screen at all